submissions/Oleksandr-Heleta/OOP_exercise/index.js

```diff
@@ -47,16 +45,20 @@ class Dog extends Animal {
 }
 
 class Cat extends Animal {
+   static say = 'moew';
    constructor(name, gender) {
-      super('cat', name, gender, "meow");
+      super('cat', name, gender, Cat.say);
+
+   }
+
+   static sayingPhrase(phrase) {
+      return Cat.say = phrase || 'meow';
    }
 }
 
 class CatWoman extends Human {
    constructor(name, gender) {
-      super(name, gender)
-      this.mutation = new Cat();
-      this.say = this.mutation.say
+      super(name, gender, Cat.say)
    }
 }
 
```

Comment:
Added static property and static method

Reply:
This is the right direction. But cat-woman's saying is still copied from a primitive.